registry/server/versioning/services/Versioning.ts

```diff
@@ -53,11 +53,18 @@ export class Versioning {
                 throw new errors.VersioningError({m: `Unable to determine changeset for entity type "${conf.type}" & ID "${conf.id}"`});
             }
 
+            const data = currentData === null ? null : JSON.stringify(currentData);
+            const data_after = newData === null ? null : JSON.stringify(newData);
+
+            if (data === data_after) {
```

Comment:
might be still not valid comparison, because 
![image](https://user-images.githubusercontent.com/1667469/172604393-a55b75bf-4659-49f6-986c-d26e3777659d.png)

Reply:
you are right, but data is received from DB, that's why the order is always the same. and thats why I decided to not complicate the codebase and comparison process.
and anyway, this condition will be used in rare cases, and if condition is used then probably the are problems on the consumer side.